lambda/custom/document/renderPage.json

```diff
@@ -53,7 +47,9 @@
             "type" : "Frame" , 
             "item" : { 
                 "type" : "Image" , 
-                "source" : "" 
+                "source" : "",
```

Comment:
Why are you leaving an empty source field? Should this be a datasource reference?

Reply:
Will add to the properties "logoURL": "", and reference it as "source" : "${payload.phrase.properties.logoUrl}",